pmlc/dialect/stripe/from_mlir.cc

```diff
@@ -474,6 +474,16 @@ std::shared_ptr<stripe::Program> FromMLIR(mlir::ModuleOp module) {
   StripeBuilder builder(func);
   auto ret = std::make_shared<stripe::Program>();
   ret->entry = builder.getResult();
+  for (const auto& ref : ret->entry->SubBlock(0)->refs) {
+    if (IsReadDir(ref.dir)) {
+      IVLOG(1, "input_shape: " << ref.from << " = " << ref.interior_shape);
```

Comment:
Maybe turn these down a bit?  (Do we have a guide somewhere for when to use which verbosity?  I've been kind of making it up as I go.)